air/src/execution_step/air/call/triplet.rs

```diff
@@ -44,30 +44,53 @@ pub(crate) fn resolve<'i>(triplet: &ast::Triplet<'i>, ctx: &ExecutionCtx<'i>) ->
 /// Resolve value to string by either resolving variable from `ExecutionCtx`, taking literal value, or etc.
 // TODO: return Rc<String> to avoid excess cloning
```

Comment:
Do these TODO still refer to correct function?  Might they need to be copied to the next one too?

Reply:
Yep, copied, thx